utils/env_test.go

```diff
@@ -39,3 +39,30 @@ func TestExpandEnvRetainingBindables(t *testing.T) {
 		})
 	}
 }
+
+func TestExpandEnvRetainingBindablesAppWideVariables(t *testing.T) {
```

Comment:
WDYT about making this test generic to the `appWideVariables` map?

I'm thinking about something like iterating over the map and constructing the in and out parameters somewhat like this (pseudo-code, written in Github so might not compile)

```golang
for var := range appWideVariables {
	t.Run(var, func(t *testing.T) {
		ref := fmt.Sprintf("${%s}", var)
		got := ExpandEnvRetainingBindables(ref)
		require.Equal(t, ref, got)
	})
}
```

That'd deal with all of them even if we add new ones.

Reply:
I feel like the values in the test case should be hard-coded, otherwise we'd be testing the _implementation_. The change you suggested will make the test pass for any value that `appWideVariables` will ever have.

IMO, the test should act as a "spec" and describe some set of product requirements.

Reply:
That's a fair take to have here indeed. Alrighty!